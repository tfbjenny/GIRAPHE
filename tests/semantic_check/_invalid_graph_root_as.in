node a = node("1");
node b = node("1");
node c = node("1");
graph g = a--b--c;
int d = 1;
g~d;