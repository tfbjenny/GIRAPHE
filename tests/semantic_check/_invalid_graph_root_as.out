invalid graph root as: g ~ d
